multimodal/src/autogluon/multimodal/data/infer_types.py

```diff
@@ -238,6 +239,46 @@ def is_image_column(
         return False
 
 
+def is_document_pdf_column(
+    data: pd.Series,
+    col_name: str,
+    sample_m: Optional[int] = 500,
+) -> bool:
+    """
+    Identify if a column is a pdf document column.
+
+    Parameters
+    ----------
+    data
+        One column of a multimodal pd.DataFrame for training.
+    col_name
+        Name of column.
+    sample_m
+        Number of sample images used to check if images are documents images.
+
+    Returns
+    -------
+    Whether the column is a pdf document column.
+    """
+
+    if len(data) > sample_m:
+        # Sample to speed-up type inference
+        data = data.sample(n=sample_m, random_state=0)
+
+    for docs in data:
+        try:
+            if not isinstance(docs, list):
+                docs = [docs]
+            for per_doc in docs:
+                # If there is non-pdf document, return False
+                if not per_doc.endswith(".pdf"):
+                    return False
```

Comment:
Just curious about a question. Do we have some logics to handle the situation in which a non-PDF document is encountered? Are there any warnings return to the users?